susemanager-utils/susemanager-sls/salt/channels/channels.repo

```diff
@@ -18,7 +18,9 @@ URIs: {{protocol}}://{{ args['token'] }}@{{hostname}}:{{port}}/rhn/manager/downl
 {%- endif %}
 Suites: {{ chan }}/
 Components:
+{%- if not salt['pillar.get']('mgr_metadata_signing_enabled', false) %}
```

Comment:
Curiosity/question: why using "not-false"? Could you have used "true" instead?
I mean, something like {%- if salt['pillar.get']('mgr_metadata_signing_enabled', true) %}
Just for readability purposes (logically it's the same)

Reply:
pillar.get(<key>, <default>) The first define the key we want to get, but if this key does not exist we can define a default value. The safe value is "false" it is not signed. But we want to set "Trusted: yes" only when metadata are not signed.

Reply:
I was missing the fact that "false" is the default value if the item is not found. That clarifies the issue. Thanks